splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/ErrorHandler.java

```diff
@@ -0,0 +1,123 @@
+package com.redhat.console.notifications.splunkintegration;
+
+import java.io.IOException;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.LoggingLevel;
+import org.apache.camel.Processor;
+import org.apache.camel.builder.endpoint.EndpointRouteBuilder;
+import org.apache.camel.http.base.HttpOperationFailedException;
+import org.apache.http.ProtocolException;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+abstract class ErrorHandler extends EndpointRouteBuilder {
```

Comment:
Maybe use name like `ErrorHandlingRouteBase`.

Reply:
ACK! Moving with `ErrorHandlingBaseRoutes` I think this name makes more sense!

Thanks